datadog_lambda/wrapper.py

```diff
@@ -105,19 +116,47 @@ def __call__(self, event, context, **kwargs):
 
     def _before(self, event, context):
         try:
+
             set_cold_start()
             submit_invocations_metric(context)
             # Extract Datadog trace context from incoming requests
-            extract_dd_trace_context(event)
+            dd_context = extract_dd_trace_context(event)
+            span_context = None
```

Comment:
Nits: Can we refactor these into a helper function rather than polluting the wrapper code with too much details?